README.md

```diff
@@ -76,6 +76,7 @@ Vendor | Device Name | CodeName | Models | Status
 Samsung | Galaxy J7 2015 | j7elte | | tested
 Samsung | Galaxy A3 2017 | a3y17lte | SM-A320FL | tested
 Samsung | Galaxy A5 2016 | [a5xelte](https://wiki.lineageos.org/devices/a5xelte) | SM-A510F | tested
+Samsung | Galaxy A5 2017 | [a5y17lte](https://doc.e.foundation/devices/a5y17lte) | | tested
```

Comment:
Wouldn't it be better to link the LineageOS wiki to stay consistent?

Reply:
Fair point! :+1: